lib/src/xdg_background_portal.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:dbus/dbus.dart';
+
+import 'xdg_portal_request.dart';
+
+/// Result of a request asking for an allowed to run in the background.
+class XdgBackgroundPortalRequestResult {
+  /// TRUE if the application is allowed to run in the background.
+  final bool background;
+
+  /// TRUE if the application is will be autostarted.
+  final bool autostart;
+
+  XdgBackgroundPortalRequestResult(
+      {required this.background, required this.autostart});
+
+  @override
+  int get hashCode => Object.hash(background, autostart);
+
+  @override
+  bool operator ==(other) =>
+      other is XdgBackgroundPortalRequestResult &&
+      other.background == background &&
+      other.autostart == autostart;
+
+  @override
+  String toString() =>
+      '$runtimeType(background: $background, autostart: $autostart)';
+}
+
+/// Portal for requesting autostart and background activity.
+class XdgBackgroundPortal {
+  final DBusRemoteObject _object;
+  final String Function() _generateToken;
+
+  XdgBackgroundPortal(this._object, this._generateToken);
+
+  /// Ask to request that the application is allowed to run in the background.
+  Stream<XdgBackgroundPortalRequestResult> requestBackground({
+    String parentWindow = '',
+    String reason = '',
```

Comment:
Make all the options arguments nullable like the other portals - sending an empty reason might be interpreted differently in the portal than not sending a reason at all.